climate-data.md

```diff
@@ -0,0 +1,37 @@
+
+# Climate open data
+
+Everyone can have access to climate data, however many people were not able to access the data due to various reasons: failing register on the website, 
+formulating and sending request to get the right data they need.
```

Comment:
Anyone can have access to climate data, however, it is difficult to gain access to the data due to various reasons:
- failing registration on the website
- formulating and sending a request to get the right data they need

Reply:
great, thank you. i agree with the changes!